frontend/index.html

```diff
@@ -12,6 +12,7 @@
       <img src="/images/face.jpg" alt="a happy face">
       <h1>This is an exapmle</h1>
     </main>
-
+    <script src="isMobile.js">
```

Comment:
this should be `script src="js/isMobile.js">`

Reply:
Sorry about that! It should work now, it worked on my end.